plugins/hello.koplugin/main.lua

```diff
@@ -1,9 +1,15 @@
+--[[--
+This is a debug plugin to test Plugin functionality
+@module koplugin.HelloWorld
```

Comment:
Reverse of the other comment, makes it seem like it's part of it when it's not.
```suggestion
This is a debug plugin to test Plugin functionality

@module koplugin.HelloWorld
```